src/core/issues-notes.js

```diff
@@ -10,178 +10,168 @@
+              const a = hyperHTML`<a href='${conf.atRiskBase + dataNum}'/>`;
+              span.before(a);
+              a.append(span);
             }
-          } else {
-            text += " " + issueNum;
-          }
-          if (report.number !== undefined) {
-            // Add entry to #issue-summary.
-            const $li = $("<li><a></a></li>");
-            const $a = $li.find("a");
-            $a.attr("href", "#" + $div[0].id).text(
-              conf.l10n.issue + " " + report.number
-            );
-            if (report.title) {
-              $li.append(
-                $(
-                  "<span style='text-transform: none'>: " +
-                    report.title +
-                    "</span>"
-                )
-              );
+            tit.querySelector("span").classList.add("issue-number");
+            ghIssue = ghIssues.get(Number(dataNum));
+            if (ghIssue && !report.title) {
+              report.title = ghIssue.title;
             }
-            $issueList.append($li);
           }
+        } else {
+          text += " " + issueNum;
         }
-        $tit.find("span").text(text);
-        if (ghIssue && report.title && githubAPI) {
-          if (ghIssue.state === "closed") $div[0].classList.add("closed");
-          const labelsGroup = Array.from(ghIssue.labels || [])
-            .map(label => {
-              const issuesURL = new URL("./issues/", conf.github.repoURL);
-              issuesURL.searchParams.set(
-                "q",
-                `is:issue is:open label:"${label.name}"`
-              );
-              return {
-                ...label,
-                href: issuesURL.href,
-              };
-            })
-            .map(createLabel)
-            .reduce((frag, labelElem) => {
-              frag.appendChild(labelElem);
-              return frag;
-            }, document.createDocumentFragment());
-          $tit.append(
-            $(
-              "<span style='text-transform: none'>: " + report.title + "</span>"
-            ).append(labelsGroup)
-          );
-          $inno.removeAttr("title");
-        } else if (report.title) {
-          $tit.append(
-            $(
-              "<span style='text-transform: none'>: " + report.title + "</span>"
-            )
-          );
-          $inno.removeAttr("title");
-        }
-        $tit.addClass("marker");
-        $div.append($tit);
-        $inno.replaceWith($div);
-        let body = $inno.removeClass(report.type).removeAttr("data-number");
-        if (ghIssue && !body.text().trim()) {
-          body = ghIssue.body_html;
+        if (report.number !== undefined) {
+          // Add entry to #issue-summary.
+          const li = hyperHTML`
+  <li>
+    <a href="${"#" + div.id}">${conf.l10n.issue + " " + report.number}</a>
+    ${
+      report.title
+        ? hyperHTML`<span style="text-transform: none">: ${report.title}</span>`
+        : ""
+    }
+  </li>`;
+          issueList.append(li);
```

Comment:
```suggestion
```

Reply:
How are the tests passing even when we have appended twice?!

Reply:
>How are the tests passing even when we have appended twice?!

Because appending a child will automatically disconnect it from the previous parent. A magic 😲

Reply:
😮 🤯